src/Data/List/Base.agda

```diff
@@ -357,17 +357,21 @@ xs ∷ʳ? x = maybe′ (xs ∷ʳ_) xs x
 
 -- Backwards initialisation
 
-infixl 5 _∷ʳ'_
+infixl 5 _∷ʳ′_
 
 data InitLast {A : Set a} : List A → Set a where
   []    : InitLast []
-  _∷ʳ'_ : (xs : List A) (x : A) → InitLast (xs ∷ʳ x)
+  _∷ʳ′_ : (xs : List A) (x : A) → InitLast (xs ∷ʳ x)
+
+infixl 5 _∷ʳ'_
+_∷ʳ'_ : (xs : List A) (x : A) → InitLast (xs ∷ʳ x)
+_∷ʳ'_ = InitLast._∷ʳ′_
```

Comment:
We usually put deprecated terms at the end of the file, just above the `WARNING_ON_USAGE ` pragma. We also don't usually include their type declaration (though it's important to keep their infix declaration.  The same goes for the other deprecated terms.

Reply:
I noticed that, but I could not get it to compile if I put the deprecated item at the end or omitted its type declaration. I am not sure why though. I can try again.

Reply:
In response, I put them at the end of the file, just above the WARNING_ON_USAGE pragma. I will try to remove the type declaration if the build succeeds.